Provider/src/main/java/com/spotify/confidence/CommonContextBuilder.kt

```diff
@@ -0,0 +1,65 @@
+package com.spotify.confidence
+
+import android.content.Context
+import android.os.Build
+import com.spotify.confidence.CommonContextKeys.APP_BUILD_KEY
+import com.spotify.confidence.CommonContextKeys.APP_NAMESPACE_KEY
+import com.spotify.confidence.CommonContextKeys.APP_NAME_KEY
+import com.spotify.confidence.CommonContextKeys.APP_VERSION_KEY
+import com.spotify.confidence.CommonContextKeys.DEVICE_MANUFACTURER_KEY
+import com.spotify.confidence.CommonContextKeys.DEVICE_MODEL_KEY
+import com.spotify.confidence.CommonContextKeys.DEVICE_NAME_KEY
+import com.spotify.confidence.CommonContextKeys.DEVICE_TYPE_KEY
+import com.spotify.confidence.CommonContextKeys.OS_NAME_KEY
+import com.spotify.confidence.CommonContextKeys.OS_VERSION_KEY
+import com.spotify.confidence.CommonContextKeys.SCREEN_DENSITY_KEY
+import com.spotify.confidence.CommonContextKeys.SCREEN_HEIGHT_KEY
+import com.spotify.confidence.CommonContextKeys.SCREEN_WIDTH_KEY
+
+internal fun Confidence.addCommonContext(context: Context): Confidence {
```

Comment:
I suggest we defer this concept for later. Please pull out of the PR but maybe store it somewhere for later inspiration.

Reply:
will be moved.